### PR TITLE
Read cache-keys.json with fs, change SERLO_ORG_HOST to API_HOST and upgrade node to 14.x

### PR DESCRIPTION
**Pull request by hugotiburtino**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -15,29 +15,39 @@
...
 void start()
 
 async function start() {
-  const pagination = process.env.PAGINATION
+  let pagination: number | undefined
+  if (process.env.PAGINATION == undefined) {
+    pagination = undefined
+  } else {
+    pagination = parseInt(process.env.PAGINATION)
+  }
```
Comment (kulla) [inline src/index.ts]:
```suggestion
  let pagination = process.env.PAGINATION === undefined ?  undefined : parseInt(process.env.PAGINATION)

```

File: src/index.ts [hunk 2]
```diff
@@ -15,29 +15,39 @@
...
   if (pagination !== undefined && pagination <= 0) {
```
Comment (kulla) [inline src/index.ts:37]:
```suggestion
  if (Number.isNaN(pagination)) {
    pagination = undefined
  }
  if (pagination !== undefined && pagination <= 0) {
```

**Review by kulla — CHANGES_REQUESTED** [review]
Just minor change requests

**Comment by hugotiburtino** [comment]
@kulla changes made

**Review by kulla — APPROVED** [review]

**Pull request merged**